### PR TITLE
Filesystem function helper

### PR DESCRIPTION
**Pull request by cclerget**
**Description of the Pull Request (PR):**

Implement few filesystem helper functions like IsDir, IsFile ...

### CODE REVIEW

File: src/pkg/util/fs/helper_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+func TestIsDir(t *testing.T) {
+	if IsDir("/etc") != true {
+		t.Errorf("IsDir returns false for directory")
+	}
+}
+
+func TestIsLink(t *testing.T) {
+	if IsLink("/proc/mounts") != true {
```
Comment (dtrudg) [inline src/pkg/util/fs/helper_test.go]: If we are going to build a client (to run remote builds) from a non-linux machine e.g. a mac then we can't expect all of these test targets to exist.

**Review by dtrudg — DISMISSED** [review]
This looks good to me, with the caveat that the tests here won't work on non-Linux OS. I'm not sure if they would ever be used if we had a remote build client on a non-Linux OS?

File: src/pkg/util/fs/helper_test.go (hunk 1 above)
Comment (cclerget) [reply src/pkg/util/fs/helper_test.go]: @dctrud Good point ! Or simply rename it to `helper_linux_test.go` ?

File: src/pkg/util/fs/helper.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package fs
```
Comment (ArangoGutierrez) [inline src/pkg/util/fs/helper.go:9]: Missing Copyright 
```go
/*
  Copyright (c) 2018, Sylabs, Inc. All rights reserved.

  This software is licensed under a 3-clause BSD license.  Please
  consult LICENSE file distributed with the sources of this project regarding
  your rights to use or distribute this software.
*/
```

File: src/pkg/util/fs/helper_linux_test.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package fs
```
Comment (ArangoGutierrez) [inline src/pkg/util/fs/helper_linux_test.go:9]: Missing Copyright
```go
/*
  Copyright (c) 2018, Sylabs, Inc. All rights reserved.

  This software is licensed under a 3-clause BSD license.  Please
  consult LICENSE file distributed with the sources of this project regarding
  your rights to use or distribute this software.
*/
```

**Review by ArangoGutierrez — DISMISSED** [review]

File: src/pkg/util/fs/helper_linux_test.go (hunk 3 above)
Comment (cclerget) [reply src/pkg/util/fs/helper_linux_test.go:9]: Good catch done !

**Review by yhcote — APPROVED** [review]
looks good to me

**Review by yhcote — APPROVED** [review]
looks good to me !

**Pull request merged**